dashbuilder-website-docs/src/main/asciidoc/guides/yaml_guide.adoc

```diff
@@ -406,10 +406,176 @@ pages:
+        dataSetLookup:
+            uuid: products
+            group:
+                - columnGroup:
+                    source: Section
+                  groupFunctions:
+                    - source: Section
+                    - source: Quantity
+                      function: JOIN
+                      column: Total Products
+----
+
+.Dataset Lookup with join
+image::guides/yaml/join.png[Lookup with group]
+
+In the example below the dataset lookup joins the number of products per section using comma:
+
+[source]
+----
+datasets:
+- uuid: products
+  content: >-
+            [
+              ["Computers", "Scanner", 5],
+              ["Computers", "Printer", 7],
+              ["Computers", "Laptop", 3],
+              ["Electronics", "Camera", 10],
+              ["Electronics", "Headphones", 5]
+            ]
+  columns:
+    - id: Section
+      type: LABEL
+    - id: Name
+      type: LABEL
+    - id: Quantity
+      type: Number
+pages:
+- components:
+    - settings:
+        dataSetLookup:
+            uuid: products
+            group:
+                - columnGroup:
+                    source: Section
+                  groupFunctions:
+                    - source: Section
+                    - source: Quantity
+                      function: JOIN_COMMA
+                      column: Total Products
+----
+
+.Dataset Lookup with join using comma
+image::guides/yaml/joinComma.png[Lookup with group]
+
+In the example below the dataset lookup joins the number of products per section using hyphen:
+
+[source]
+----
+datasets:
+- uuid: products
+  content: >-
+            [
+              ["Computers", "Scanner", 5],
+              ["Computers", "Printer", 7],
+              ["Computers", "Laptop", 3],
+              ["Electronics", "Camera", 10],
+              ["Electronics", "Headphones", 5]
+            ]
+  columns:
+    - id: Section
+      type: LABEL
+    - id: Name
+      type: LABEL
+    - id: Quantity
+      type: Number
+pages:
+- components:
+    - settings:
+        dataSetLookup:
+            uuid: products
+            group:
+                - columnGroup:
+                    source: Section
+                  groupFunctions:
+                    - source: Section
+                    - source: Quantity
+                      function: JOIN_HYPHEN
+                      column: Total Products
+----
+
+.Dataset Lookup with join using hyphen
+image::guides/yaml/joinHyphen.png[Lookup with group]
+
+The supported group functions are `SUM`, `MAX`, `MIN` , `AVERAGE` and `MEDIAN` for numbers. For label columns the supported functions are `DISTINCT` and `COUNT`.  
 By default it uses the column name itself, it is also possible to give another name to the grouped column.
 If a column is not specific in columnGroup, but used with other columns in columnFunctions, then the error  **Error during dataset lookup: No aggregation function specified for the column** is displayed.
 
+New group functions are as follows:-
```

Comment:
Please remove the word "new". In the documentation a reference to new may confuse users. I would use something like: "Here's a summary of all supported group functions". Then please include the other group functions in your list.